src/embezzler.ts

```diff
@@ -912,11 +912,29 @@ export function estimatedTurns(): number {
   const thumbRingMultiplier = usingThumbRing() ? 1 / 0.96 : 1;
 
   // We need to estimate adventures from our organs if we are only dieting after yachtzee chaining
-  const fullnessAdventures = (fullnessLimit() - myFullness()) * 8;
-  const inebrietyAdventures = (inebrietyLimit() - myInebriety()) * 7;
+  const distentionPillSpace = have($item`distention pill`) && !get("_distentionPillUsed") ? 1 : 0;
+  const syntheticPillSpace =
+    have($item`synthetic dog hair pill`) && !get("_syntheticDogHairPillUsed") ? 1 : 0;
+  const shotglassSpace = have($item`mime army shotglass`) && !get("_mimeArmyShotglassUsed") ? 1 : 0;
+  const sweatSpace = have($item`designer sweatpants`) ? 3 - get("_sweatOutSomeBoozeUsed") : 0;
+  const fullnessAdventures = (fullnessLimit() - myFullness() + distentionPillSpace) * 8;
+  const inebrietyAdventures =
+    (inebrietyLimit() - myInebriety() + syntheticPillSpace + sweatSpace + shotglassSpace) * 7;
+  const borrowedTimeAdventures = globalOptions.ascending && !get("_borrowedTimeUsed") ? 20 : 0;
+  const chocolateAdventures = ((3 - get("_chocolatesUsed")) * (4 - get("_chocolatesUsed"))) / 2;
+  const yachtzeeTurns = 30; // guesstimate
+  const bufferAdventures = 30; // We don't know if garbo would decide to use melange/voraci tea/sweet tooth to get more adventures
   const adventuresAfterChaining =
     globalOptions.yachtzeeChain && !get("_garboYachtzeeChainCompleted")
-      ? Math.max(fullnessAdventures + inebrietyAdventures - 30, 0)
+      ? Math.max(
+          fullnessAdventures +
+            inebrietyAdventures +
+            borrowedTimeAdventures +
+            chocolateAdventures +
+            bufferAdventures -
+            yachtzeeTurns,
+          0
+        )
```

Comment:
Could we slice all of the various organ space predictors into another function so that when we're estimating turns it appears as one big thing?

Maybe also slice out the chocolate and borrowed item stuff?

After that, add double-spacing in places where the vibe feels right to make it easier to follow.

Really, just SLAP it a bit.